syntax="proto2";

import "aap_protobuf/service/control/message/VoiceSessionStatus.proto";

package aap_protobuf.service.control.message;

message VoiceSessionNotification
{
    optional VoiceSessionStatus status = 1;

}